code/modules/mob/living/carbon/human/species_types/beeperson.dm

```diff
@@ -0,0 +1,82 @@
+/datum/species/human/beepeople
```

Comment:
reskinned human confirmed????

Reply:
they are kinda like catpeople. they dna is just a little of human, a little of bee and a ton of unestable mutadone, grind for 5 minutes and you have it